examples/README.md

```diff
@@ -21,6 +25,10 @@ some  particular aspect of Bokeh capability (such as [linking and
 brushing](http://www.infovis-wiki.net/index.php?title=Linking_and_Brushing)),
 or walk through a particular example in additional detail.
 
+### [`Integration`](integration/)
+
+This directory contains code useful with Bokeh's integrations.
```

Comment:
The examples in the Integration subdirectory are all specifically crafted for image-diff integration tests. They are in this location as a technical compromise, but mostly are probably are not interesting/useful to users. I would suggest simply omitting mention of this directory of the README altogether

Reply:
Exactly; integration examples are on they way out, so there is no point in mentioning them anymore.

Reply:
Yess, thank you for the suggestion, I'll update the file